valid.js

```diff
@@ -0,0 +1,27 @@
+const form = document.querySelector('.contact-form');
```

Comment:
- [ ] You have done a good job with the error message :muscle: but there is an issue because when I try submitting the form with the correct information the form still shows the same error message and the "Get In Touch" button does not seems to work. Please, kindly fix this issue for a better user experience :wink:
![image](https://user-images.githubusercontent.com/81522435/146197005-7dce8d7a-1bf2-47de-947a-617ed1c3cdf0.png)
**You could see the `outline` in the button because it's in `active` state since it has been pressed** :pushpin: